activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -812,7 +816,8 @@ def column_definitions(table_name)
           query(<<~SQL, "SCHEMA")
               SELECT a.attname, format_type(a.atttypid, a.atttypmod),
                      pg_get_expr(d.adbin, d.adrelid), a.attnotnull, a.atttypid, a.atttypmod,
-                     c.collname, col_description(a.attrelid, a.attnum) AS comment
+                     c.collname, col_description(a.attrelid, a.attnum) AS comment,
+                     #{supports_virtual_columns? ? 'attgenerated' : quote('')} as attgenerated
```

Comment:
In PG 12, do non-generated columns return an empty string here (`quote('')`) or NULL?

Reply:
I tested on our PG12 server, and it looks like an empty string (pgAdmin will show `[null]` for NULL values):
<img width="130" alt="Screen Shot 2020-10-01 at 3 55 49 PM" src="https://user-images.githubusercontent.com/1305439/94871380-b2b95900-03fe-11eb-936d-ac9b2e746ca9.png">